src/main.ts

```diff
@@ -17,6 +17,9 @@ console.log("🚀 ~ User operation Completed:", status)
 };
 
 export const Cypher = async ({address, targetChainIdHex: fromChainId, requiredTokenContractAddress: fromTokenContractAddress, requiredTokenBalance, isTestnet, callBack = noop }: DappDetails): Promise<void> => {
+  if (screen.width < 768) {
```

Comment:
Are we disabling the feature on mobile here?

Reply:
Yes. Currently the popup won't show if the screen width is < 768 pixels. which is typically for mobiles.